src/test/java/uk/nhs/tis/trainee/usermanagement/service/UserAccountServiceIntegrationTest.java

```diff
@@ -0,0 +1,132 @@
+/*
+ * The MIT License (MIT)
+ *
+ * Copyright 2023 Crown Copyright (Health Education England)
```

Comment:
You're living in the past mate 😁

Reply:
I'm not well up on my copyright laws, given this is lifted and unchanged from another service as written in 2023... do we update the copyright here or leave it then same 🤔